static/styles/zulip.scss

```diff
@@ -1090,13 +1090,35 @@ a.message_label_clickable:hover {
     opacity: 1.0;
 }
 
-.on_hover_topic_mute {
+.on_hover_topic_mute,
+.on_hover_topic_read,
+.on_hover_topic_bookmark {
     opacity: 0.2;
+    &:hover {
+        cursor: pointer;
+        opacity: 1.0;
+    }
 }
 
-.on_hover_topic_mute:hover {
-    cursor: pointer;
-    opacity: 1.0;
+.bookmarked {
+    opacity: 1.0 !important;
+}
+
+#recent_topics_filter_buttons {
+    padding: 10px;
+}
+
+#recent_topics_search {
+    margin-left: 20px !important;
+    width: 350px !important;
+}
+
+.btn-recent-filters {
+    border-radius: 40px;    
+}
+
+.btn-recent-selected {
+    background-color: hsl(0, 4%, 80%) !important;
 }
```

Comment:
I think we should namespace this `recent_topics` specific CSS inside `recents.scss`.